modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLConfiguration.java

```diff
@@ -34,6 +34,9 @@
 	@Meta.AD(deflt = "true", name = "add-default-structures", required = false)
 	public boolean addDefaultStructures();
 
+	@Meta.AD(deflt = "15", name = "check-interval", required = false)
```

Comment:
@adolfopa I do not know it this can be a impediment for the release or can give us some problems.
if can be a problem we can omit the listener